From 1c14a55c60417a5e7f70151f938284a5e36cff17 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Wed, 20 Nov 2024 15:41:14 +0100
Subject: [PATCH 1357/1695] dts: stm32: remove vdd33ucpd on stm32mp215f-dk

There is no vdd33ucpd in STM32MP21.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 995674100d64ce9ccedc1fa4aee9bdbf678e8c77 dts: stm32: remove vdd33ucpd on stm32mp215f-dk]
Change-Id: Ib9290cd69217a19c66f750af1bc94bccc331ffbb
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419811
Tested-by: Pascal PAILLET <p.paillet@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 7b0a92c23..b472290cd 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -284,13 +284,6 @@
 	status = "okay";
 	wakeup-gpios = <&gpioa 0 GPIO_ACTIVE_HIGH>, <0>, <0>, <0>, <0>, <0>;
 
-	vdd33ucpd {
-		status = "okay";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
-		vdd33ucpd-supply = <&vdd3v3_usb>;
-	};
-
 	vdda18adc {
 		status = "okay";
 		regulator-min-microvolt = <1800000>;
-- 
2.47.2

